NYT.py

```diff
@@ -0,0 +1,213 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC ###Downloading libraries
+
+# COMMAND ----------
+
+!pip install kaggle
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ###Downloading the NYT dataset
+
+# COMMAND ----------
+
+import os
+
+dbutils.fs.cp('/FileStore/kaggle.json', 'file:/kaggle/kaggle.json')
+os.environ['KAGGLE_CONFIG_DIR'] = '/kaggle'
+
+!kaggle datasets download -d aryansingh0909/nyt-articles-21m-2000-present --force
+!unzip nyt-articles-21m-2000-present.zip
+
+dbutils.fs.mv("file:/databricks/driver/nyt-metadata.csv", "/FileStore/nyt-metadata.csv")
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ###Reading .csv file to DataFrame
+
+# COMMAND ----------
+
+from pyspark.sql.types import StructField, StructType, StringType, TimestampType, IntegerType, FloatType
+
+nyt_schema = StructType([
+    StructField('abstract', StringType(), True), 
+    StructField('web_url', StringType(), False), 
```

Comment:
be careful with those False values - during reading spark does not actually restrict inserting null values